src/server/cli/test-runner.js

```diff
@@ -121,6 +121,13 @@ async function getTestRunnerOptions(argv = []) {
 			description: 'Run tests in Webkit',
 			order: 4
 		},
+		{
+			name: 'test-reporting',
+			type: Boolean,
+			defaultOption: false,
+			description: 'Enables generating a D2L Test Reporting format JSON file',
+			order: 14
+		}
```

Comment:
Would it make sense to just always enable reporting and either not make it an option, or enable it by default?

Reply:
I was thinking of this but wanted to try out a couple of integrations first. Then we can have it default to always generate.